bertopic/_save_utils.py

```diff
@@ -0,0 +1,315 @@
+import os
+import sys
+import json
+import numpy as np
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+
+# HuggingFace Hub
+try:
+    from huggingface_hub import (
+        create_repo, get_hf_file_metadata,
+        hf_hub_download, hf_hub_url,
+        repo_type_and_id_from_hf_id, upload_folder)
+    _has_hf_hub = True
+except ImportError:
+    _has_hf_hub = False
+
+# Typing
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+from typing import Union, Mapping, Any
+
+# Safetensors check
+try:
```

Comment:
Would it be worth doing this check inside the save function? For the user who is running in Colab or another notebook environment, they might hit the assertion error about safetensors when they try and save a model, but since the import check is done earlier, they would need to reload the module even after they pip install safetensors.

Reply:
Thanks, will definitely do this. It is indeed something users will run into since `safetensors` is not (yet) part of the transformers base requirements.